client/my-sites/plan-features-2023-grid/plan-comparison-grid.tsx

```diff
@@ -355,6 +370,9 @@ export const PlanComparisonGrid: React.FC< PlanComparisonGridProps > = ( {
 	const isSmallBreakpoint = usePricingBreakpoint( 880 );
 
 	const [ visiblePlans, setVisiblePlans ] = useState< string[] >( [] );
+	const [ visibleFeatureGroups, setVisibleFeatureGroups ] = useState< string[] >( [
+		Object.keys( featureGroupMap )[ 0 ],
```

Comment:
Nit: This 0 index is a bit hard to understand, maybe getting it into a variable might help.
```javascript
const [ firstSetOfFeatures ] = Object.keys( featureGroupMap );
```

Reply:
Thanks! this makes it look clearer.